src/components/common/chat-popup.vue

```diff
@@ -67,10 +67,12 @@ export default class ChatPopup extends Vue {
     if (this.webChatStatusUrl) {
       this.chatStatus = await axios
         .get(this.webChatStatusUrl)
-        .then(response => (this.chatStatus = response.data.status))
+        .then(response => {
+          return response.data?.status || 'response error'
+        })
         .catch(error => {
           console.error('failed to get webchat status, error =', error) // eslint-disable-line no-console
-          this.chatError = true
+          return error.response?.statusText || 'network error'
```

Comment:
do we need to call this every so often in case someone is sitting on this page?

Reply:
Probably a good idea but not part of the requirements at this time :/

Reply:
https://github.com/bcgov/entity/issues/7031#issuecomment-815195154